contract/src/test/kotlin/io/provenance/scope/loan/contracts/UpdateENoteContractUnitTest.kt

```diff
@@ -0,0 +1,28 @@
+package io.provenance.scope.loan.contracts
+
+import io.kotest.core.spec.style.WordSpec
+
+class UpdateENoteContractUnitTest : WordSpec({
+    "updateENote" When {
+        "given an empty input" xshould {
+            "throw an appropriate exception" {
+                // TODO: Implement
+            }
+        }
+        "given an invalid input" xshould {
+            "throw an appropriate exception" {
+                // TODO: Implement
+            }
+        }
+        "given an input which attempts to silently change properties of the existing eNote" xshould {
```

Comment:
"silently change" == "provide the same checksum but alter other fields", see `documentModificationValidation`.